src/v/redpanda/admin/transform.cc

```diff
@@ -20,6 +20,8 @@
 #include "redpanda/admin/util.h"
 #include "transform/api.h"
 
+#include <system_error>
```

Comment:
just curious - why move from `cluster::errc` to `std::error_code` here?

Reply:
I think the commit message on 4c4b9f66121399c9a68d2366baadc52f68942ffd answers my question

Reply:
Exactly, we need to work with error codes from multiple subsystems now.